### DISCUSSION TITLE
考虑在角色介绍页面增加技能介绍吗

### DISCUSSION THREAD

**Discussion by ID114514** [Arondight/Adachi-BOT#664] [Ideas]
命座介绍感觉很鸡肋，了解角色更多的是技能吧

**Comment by mark9804** [comment]
不会添加，这和我们认为的角色介绍模块的功能不一样。如果觉得命座介绍鸡肋的话，所有的生日声优神之眼信息都是鸡肋中的鸡肋。

我们默认用户知道角色技能大概是什么样的（事实上就平A，E和Q，这些都是游戏中通过直观的视觉和听觉信息反馈给你的，比命座信息这种纯文本更容易产生记忆痕迹），由我们来补充用户印象不深刻或者根本记不清的东西。

**Comment by mark9804** [comment]
不过倒是可以把基础防御加上

**Reply by mark9804** [reply]
祈愿统计基于用户提供的 session id，属于私人信息，因此和 #361 同理，不会做支持
